roles/validations/tasks/security/invoke_tlse_playbooks.yml

```diff
@@ -1,11 +1,52 @@
-- name: Invoke the automation playbook (dataplane certs) on controller-0
+- name: Load CI framework parameters
+  ansible.builtin.include_vars:
+    dir: "{{ cifmw_basedir }}/artifacts/parameters"
+
+- name: Define zuul variable with project paths
+  ansible.builtin.set_fact:
+    required-projects:
```

Comment:
it is wrong place to put required-projects. It should be in Zuul job definition.